src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -31,6 +31,8 @@ namespace System.Collections
 #if FEATURE_NETCORE
     [FriendAccessAllowed]
 #endif
+    /// <summary>Implements the <see cref="T:System.Collections.IList" /> interface using an array whose size is dynamically increased as required.To browse the .NET Framework source code for this type, see the Reference Source.</summary>
```

Comment:
Can we change the tool to have a max length on the content, such that beyond that it uses multiple lines and wraps, e.g. this should be more like:
```C#
/// <summary>
/// Implements the <see cref="T:System.Collections.IList" /> interface using an array whose size is 
/// dynamically increased as required.To browse the .NET Framework source code for this type, see the 
/// Reference Source.
/// </summary>
```